pkg/operator2/configsync.go

```diff
@@ -12,6 +12,14 @@ import (
 	"github.com/openshift/library-go/pkg/operator/resourcesynccontroller"
 )
 
+type SyncResourceType string
```

Comment:
Follow the pattern in #83 and set up a single static sync - there is no flexibility on the name required.

Reply:
Done, the new spot is much better indeed.